hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestStorageContainerManager.java

```diff
@@ -180,7 +181,7 @@ private void testRpcPermissionWithConf(
 
       try {
         ContainerWithPipeline container2 = mockClientServer
-            .allocateContainer(xceiverClientManager.getType(),
+            .allocateContainer(SCMTestUtils.getReplicationType(ozoneConf),
             HddsProtos.ReplicationFactor.ONE,  "OZONE");
```

Comment:
"OZONE" could be replaced by OzoneConsts.OZONE if it's not required to be in capital letters.
Otherwise, we can add a new constant in OzoneConsts.